src/components/Channel/ChannelSide.js

```diff
@@ -0,0 +1,127 @@
+import React, { useState, useEffect } from 'react';
+import useSWR from 'swr';
+import styled from '@emotion/styled';
+import Button from '../Button';
+
+async function fetcher() {
+  const baseUrl = process.env.NEXT_PUBLIC_API_URL;
+  const channelId = '6135b03f428aabe0cf791289';
+
+  try {
+    const response = await fetch(`${baseUrl}/channel/${channelId}`, {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    });
+    const { data } = await response.json();
+
+    return data;
+  } catch (err) {
+    console.error(err);
+  }
+}
+
+export default function ChannelSide() {
+  const [currentUsers, setCurrentUsers] = useState([]);
+  const [players, setPlayers] = useState(null);
+  const [audience, setAudience] = useState(null);
+  const [host, setHost] = useState(null);
+  const { data: channel } = useSWR('/channel', fetcher);
+
+  useEffect(() => {
+    if (!channel) return;
+
+    const { players, audience, host } = channel;
+
+    setPlayers(players);
+    setAudience(audience);
+    setHost(host);
+  }, [channel]);
+
+  useEffect(() => {
+    const userId = JSON.parse(sessionStorage.getItem('user'));
+
+    setCurrentUsers(currentUsers.concat(userId));
+  }, []);
+
+  return (
+    <SideContainer>
+      <PlayerWrapper>
+        <div className="player-profile-list">
+          {players?.map((player) => {
+            return (
+              <div key={player.userId} className="player-profile">
+                <div className="player">{player.userId}</div>
+                <div className="character">{player.characterId}</div>
+              </div>
+            );
+          })}
```

Comment:
```suggestion
          {players?.map(player => (
              <div key={player.userId} className="player-profile">
                <div className="player">{player.userId}</div>
                <div className="character">{player.characterId}</div>
              </div>
            ))}
```
중괄호와 return을 쓰지 않고 소괄호로 감싸는 경우에는 return을 넣어주지 않아도 된답니다 꿀팁! 😀